test/unit/count_test.rb

```diff
@@ -15,7 +15,7 @@ class CountTest < Test::Unit::TestCase
       should "count all documents by the leaving index empty" do
         c = Search::Count.new
         assert c.indices.empty?, "#{c.indices.inspect} should be empty"
-        assert_match %r|localhost:9200/_count|, c.url
+        assert_match %r|foobarbaz:9200/_count|, c.url
```

Comment:
do we want this? should we change it to `ENV["ELASTICSEARCH_URL"]`?

Reply:
Also, where does this foobarbaz env var come from? is that set in bigmaven?

Reply:
yea its part of me trying to fix the test suite